admin/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -0,0 +1,436 @@
+[id='proc-rhdh-deploy-eks-using-helm_{context}']
+= Deploying {product} on Elastic Kubernetes Services (EKS) using Operator
+
+You can deploy the {product-short} on EKS using the Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework].
```

Comment:
This description seems ok for, 
4.2.1. Installing the Operator with OLM framework
4.2.2. Installing the Operator without OLM framework

Should we add some text here that should also cover the section: 4.2.3. Installing the Developer Hub instance in EKS? What do you think?